rr_custom_msgs/msg/ArmControl.msg

```diff
@@ -0,0 +1 @@
+float32[6] arm_velocities
```

Comment:
To make this as future proof as possible I would suggest using [JointState](http://docs.ros.org/en/melodic/api/sensor_msgs/html/msg/JointState.html) message instead. Don't know what's the current state of this though, so it might not be desirable.

Reply:
@msadowski  It's not desirable at this time as the Arm doesn't use the units the JointState message does so will probably get confusing when debugging